dhall/dhall.cabal

```diff
@@ -22,6 +22,8 @@ Description:
     .
     Read "Dhall.Tutorial" to learn how to use this library
 Category: Compiler
+Extra-Doc-Files:
```

Comment:
Should this be in `Extra-Source-Files`?  My understanding is that `Extra-Doc-Files` is intended primarily for assets for the generated Haddock documentation (e.g. images)

Reply:
Ahh, okay. I will move it.